lumen/ai/coordinator.py

```diff
@@ -82,6 +82,17 @@ class Coordinator(Viewer):
     suggestions = param.List(default=GETTING_STARTED_SUGGESTIONS, doc="""
         Initial list of suggestions of actions the user can take.""")
 
+    prompt_templates = param.Dict(default={
```

Comment:
Should we factor out a baseclass for Agent and Coordinator with shared methods and parameters?